app/ai/voice/agents/breeze_buddy/services/telephony/twilio/conference.py

```
@@ -0,0 +1,414 @@
+                    friendly_name=conference_name, status="in-progress", limit=1
+                )
+                
+                if conferences:
+                    conference_sid = conferences[0].sid
+                    logger.info(f"Conference SID retrieved: {conference_sid} (attempt {attempt + 1})")
+                    return conference_sid
+                
+                logger.debug(f"Conference not found yet (attempt {attempt + 1}/{max_retries})")
+                time.sleep(retry_delay)
+                
+            except Exception as list_error:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)